tcp/session.go

```diff
@@ -175,37 +175,37 @@ func (s *Session) handleBlockwise(w *ResponseWriter, r *pool.Message) {
 	s.handler(w, r)
 }
 
-func (s *Session) handleSignals(w *ResponseWriter, r *pool.Message) {
+func (s *Session) handleSignals(r *pool.Message) bool {
 	switch r.Code() {
 	case codes.CSM:
 		if s.disablePeerTCPSignalMessageCSMs {
-			return
+			return true
 		}
 		if size, err := r.GetOptionUint32(coapTCP.MaxMessageSize); err == nil {
 			atomic.StoreUint32(&s.peerMaxMessageSize, size)
 		}
 		if r.HasOption(coapTCP.BlockWiseTransfer) {
 			atomic.StoreUint32(&s.peerBlockWiseTranferEnabled, 1)
 		}
-		return
+		return true
 	case codes.Ping:
 		if r.HasOption(coapTCP.Custody) {
 			//TODO
 		}
-		s.sendPong(w, r)
-		return
+		s.sendPong(r.Token())
+		return true
 	case codes.Release:
 		if r.HasOption(coapTCP.AlternativeAddress) {
 			//TODO
 		}
-		return
+		return true
 	case codes.Abort:
 		if r.HasOption(coapTCP.BadCSMOption) {
 			//TODO
 		}
-		return
+		return true
 	}
-	s.handleBlockwise(w, r)
+	return false
```

Comment:
What about single `default: return false` and here return true? It would be more readable.